com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -2510,24 +2542,25 @@ public override void OnDestroy()
         }
 
         /// <inheritdoc/>
-        public override void OnGainedOwnership()
+        public override void OnLostOwnership()
```

Comment:
Leaving these here otherwise the validation test will fail even though a user can still override both methods (see OnGainedOwnership too).